plutus-tx/src/PlutusTx/Blueprint/TH.hs

```diff
@@ -83,12 +110,7 @@ mkSchemaClause ts ctorIndexes =
     let whereDecls = []
     TH.clause patterns (TH.normalB body) whereDecls
 
-  mkSchemaConstructor :: (TH.ConstructorInfo, Natural) -> TH.ExpQ
-  mkSchemaConstructor (TH.ConstructorInfo{..}, naturalToInteger -> ctorIndex) = do
+  mkSchemaConstructor :: (TH.ConstructorInfo, SchemaInfo, Natural) -> TH.ExpQ
```

Comment:
where does the info for the type go? does it do nothing but apply to the constructors? write it down!

Reply:
Done.